homeassistant/components/yale_smart_alarm/lock.py

```diff
@@ -47,12 +47,14 @@ def __init__(
         super().__init__(coordinator, data)
         self._attr_code_format = f"^\\d{code_format}$"
 
-    async def async_unlock(self, **kwargs) -> None:
+    async def async_unlock(self, **kwargs: str | None) -> None:
```

Comment:
```suggestion
    async def async_unlock(self, **kwargs: Any) -> None:
```
Looking at other intergrations this typed as `Any`, I don't think you can guaranty it will be a `str`.

Reply:
possibly so I'll change this